tasks-schema.json

```diff
@@ -142,10 +159,46 @@
                         "optional"
                       ]
                     },
+                    "target_date": {
+                      "description": "An object containing arrays of required and optional unique target dates. For short-term forecasts, the target_date specifies the date of occurrence of the outcome of interest. For instance, if models are requested to forecast the number of hospitalizations that will occur on 2022-07-15, the target_date is 2022-07-15",
+                      "example": "2022-11-05",
+                      "type": "object",
+                      "properties": {
+                        "required": {
+                          "description": "Array of target date unique identifiers that must be present for submission to be valid. Can be null if no target dates are required and all valid target dates are specified in the optional property.",
+                          "type": [
+                            "array",
+                            "null"
+                          ],
+                          "items": {
+                            "type": "string",
+                            "format": "date"
+                          }
+                        },
+                        "optional": {
+                          "description": "Array of valid but not required unique target date identifiers. Can be null if all target dates are required and are specified in the required property.",
+                          "type": [
+                            "array",
+                            "null"
+                          ],
+                          "items": {
+                            "type": "string",
+                            "format": "date"
+                          }
+                        }
+                      },
+                      "required": [
+                        "required",
+                        "optional"
+                      ]
+                    },
```

Comment:
Is this "target_date" information required? My understanding is that it would be redundant with "origin_date" and "horizon".

Reply:
My understanding from the documentation on the [Task ID variables](https://hubdocs.readthedocs.io/en/latest/format/intro-data-formats.html#task-id-variables) in the Data Formats Overview page where `target_date` is listed as a `task_id` is that there are a few ways to specify target date and the option to include it directly is acceptable? The docs also include the following note:

> We encourage hubs to avoid redundancy in the model task columns. For example, Hubs should not include all three of target_date, origin_date, and horizon as task id columns because if any two are specified, the third can be calculated directly. Similarly, if a variable is constant, it should not be included. For example, if a Hub doesn’t include multiple outcome variables, an outcome variable should not be included among the task id columns. 

Perhaps a decision was made that isn't reflected in the docs?

Reply:
I think target_date should not be required but should be one of several optional, valid ways to specify dates.